pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -712,7 +703,7 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 		klog.V(verbosity).InfoS(detailedMsg, "pod", klog.KObj(volumeToMount.Pod))
 		resizeOptions.DeviceMountPath = volumeMounter.GetPath()
 
-		_, resizeError = og.nodeExpandVolume(volumeToMount, actualStateOfWorld, resizeOptions)
+		_, resizeError = og.expandVolumeDuringMount(volumeToMount, actualStateOfWorld, resizeOptions)
```

Comment:
are you changing the function name from nodeExpandVolume to expandVolumeDuringMount? The tests should also updated?